helper/linker.go

```diff
@@ -43,70 +62,139 @@ func (f FileLinker) Execute() (map[string]string, error) {
 		layerDir = "/layers/paketo-buildpacks_open-liberty/open-liberty-runtime"
 	}
 
-	b, ok, err := bindings.ResolveOne(f.Bindings, bindings.OfType("open-liberty"))
-	if err != nil {
-		return nil, fmt.Errorf("error resolving bindings: %w", err)
+	if err = f.Configure(layerDir, appDir); err != nil {
+		return nil, fmt.Errorf("error during configuration: %w", err)
 	}
+	return nil, nil
```

Comment:
```suggestion
	return map[string]string{}, nil
```

I don't think it matters a ton, but from a style standpoint, we try to return empty values.